src/pages/partners/_partnership-opportunities.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { Header, Text, CardStyle } from 'components/elements'
+import { localize, LocalizedLink } from 'components/localization'
+import { Container, SectionContainer, Flex } from 'components/containers'
+// import device from 'themes/device'
+// Icons
+import PartnerAffiliate from 'images/svg/partner-affiliate.svg'
+import PartnerPaymentAgent from 'images/svg/partner-payment-agent.svg'
+
+const StyledSection = styled(SectionContainer)``
+const StyledFlex = styled(Flex)``
```

Comment:
what is the purpose of using `StyledSection` and `StyledFlex` when we didn't add any css? we can use `SectionContainer` and `Flex` instead.

Reply:
thanks, let me remove them